lib/live_cluster/client/cluster.py

```diff
@@ -453,19 +453,20 @@ def get_node(self, node) -> list[Node]:
 
     def get_nodes(
         self,
-        nodes: Union[
-            Literal["all"], Literal["random"], Literal["principal"], list[str]
-        ],
+        nodes: Literal[constants.NodeSelection.ALL]
+        | Literal[constants.NodeSelection.PRINCIPAL]
+        | Literal[constants.NodeSelection.RANDOM]
+        | list[str],
```

Comment:
nit: I think you can reduce this by putting all the values in a single `Literal[...]`. https://peps.python.org/pep-0586/#shortening-unions-of-literals

```suggestion
        nodes: Literal[constants.NodeSelection.ALL, constants.NodeSelection.PRINCIPAL, constants.NodeSelection.RANDOM] | list[str],
```